lib/rubocop/cop/lint/missing_super.rb

```diff
@@ -60,6 +67,22 @@ module Lint
       #     end
       #   end
       #
+      # @example AllowedParentClasses: [MyAbstractClass]
+      #   # good
+      #
+      #   class MyConcreteClass < MyAbstractClass
+      #     def initialize
+      #       # do_something without calling super
+      #     end
+      #   end
+      #
+      #   class ClassWithNoParent
+      #     def initialize
+      #       # there's no need to specify Object or BasicObject
+      #       # in the AllowedParentClasses option.
+      #     end
+      #   end
```

Comment:
The example `ClassWithNoParent` class is irrelevant to the `AllowedParentClasses` configuration and is always allowed, so it looks better to put it in the `# good` case of `@example`, right?

Reply:
Of course, I can move it there, but I wanted to make it absolutely clear that no matter what the value of `AllowedParentClasses` is, `Object` and `BasicObject` will always be allowed, so this example is just to remark that one more time in case the reader misses it from the description above. Although it can technically go in both places, I feel like it's more useful here. WDYT?

Reply:
Yeah, I understand the intention. But `@example ConfigurationName` is typically used to describe unique behaviors of `ConfigurationName`. For the sake of consistent documentation structure, it would be better to move this to `@example` I think :-)

Reply:
Can't really argue with that, I understand the value of consistency 🙂 
I'll move this up as suggested 👍